File: suites/05_ml_primitives/comparisons/convolution.py
def conv_score(w: int, h: int) -> int:
    total = 0
    for y in range(1, h - 1):
        for x in range(1, w - 1):
            total += ((x * 3) + (y * 5)) % 97
    return total
if __name__ == "__main__":
    print(conv_score(512, 512))

File: suites/05_ml_primitives/comparisons/convolution.cpp
#include <cstdio>
static long long conv_score(long long w, long long h) {
    long long total = 0;
    for (long long y = 1; y < h - 1; ++y)
        for (long long x = 1; x < w - 1; ++x)
            total += ((x * 3) + (y * 5)) % 97;
    return total;
}
int main() { printf("%lld\n", conv_score(512, 512)); return 0; }
